Assets/Scripts/Actors/ActorComponents/PlayerActorInventory.cs

```diff
@@ -88,29 +111,26 @@ public void CheckUseItem( RaycastHit hitInfo )
 
 	void CheckGiveResources( RaycastHit hitInfo )
 	{
-		if ( hitInfo.transform )
-		{
-			BuddyStats buddyStats = hitInfo.transform.GetComponent<BuddyStats>();
+		BuddyStats buddyStats = hitInfo.transform.GetComponent<BuddyStats>();
```

Comment:
Do we know that hitInfo will always have content when it gets here? Going to run into errors otherwise